cmd/manager/main.go

```diff
@@ -49,52 +50,76 @@ var (
 )
 
 func init() {
-	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+	if err := clientgoscheme.AddToScheme(scheme); err != nil {
+		panic(err)
+	}
 
-	podNamespace, ok := os.LookupEnv("POD_NAMESPACE")
```

Comment:
Configuration loading should typically happen somewhere below `main`, with `init` being as small as possible. This was removed to reduce the size of `init`, and put that loading in a more idiomatic location.

Notably, this also removes some global variables (like `namespace`), which are also best avoided when possible.

Reply:
is there a reason we keep the schema setup in the init vs the main? My vague understanding of init is that you use it mostly for running code whenever a package is installed (like i think pulling in a mysql package has an init that registers the driver or something so its ready before you use it?). In our case, its a main package, so i wonder if it should be used at all?

Reply:
Done